smoke-test/tests/cypress/cypress/integration/mutations/add_users.js

```diff
@@ -22,8 +22,9 @@ describe("add_user", () => {
 
         cy.clickOptionWithText("Invite Users")
 
-        cy.waitTextVisible('signup?invite_token').then(($elem) => {
+        cy.waitTextVisible(/signup\?invite_token=\w+/).then(($elem) => {
```

Comment:
Sometimes the UI was not loading `signup?invite_token=` without any token and that got copied which led to invalid link. So adding regex that there should be something at the end of `=`